### PR TITLE
add ut for binding/common

### PR DESCRIPTION
**Pull request by AllenZMC**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
add ut for binding/common
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RainbowMango** [comment]
cc @Poor12 Please help with it.

File: pkg/controllers/binding/common_test.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+					Name:     "foo",
+					Replicas: 2,
+				},
+			},
+			want: map[string]int64{
+				"foo": 2,
+			},
+		},
+		{
+			name: "different cluster",
```
Comment (Poor12) [inline pkg/controllers/binding/common_test.go:116]: different clusters

File: pkg/controllers/binding/common_test.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+					Name:     "foo",
+					Replicas: 1,
+				},
+			},
+			want: map[string]int64{
+				"foo": 1,
+			},
+		},
+		{
+			name: "same cluster",
```
Comment (Poor12) [inline pkg/controllers/binding/common_test.go:100]: the same cluster

File: pkg/controllers/binding/common_test.go (hunk 1 above)
Comment (AllenZMC) [reply pkg/controllers/binding/common_test.go]: Fixed. @Poor12

File: pkg/controllers/binding/common_test.go [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+				},
+				{
+					Name:     "foo",
+					Replicas: 2,
+				},
+			},
+			want: map[string]int64{
+				"foo": 2,
+			},
+		},
```
[the remark this answers is not in the document]
Comment (AllenZMC) [reply pkg/controllers/binding/common_test.go]: Fixed. @RainbowMango

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**